submission/TaylorGP/metadata.yml

```diff
@@ -0,0 +1,14 @@
+authors: # the participants
+  Baihe He, Yuxiang Gao, Qiang Lu. 
+email: hebaihe@hotmail.com
+name: TaylorGP # name of the submission /method
+description: | # anything you'd like here to describe the method.
+  Taylor Genetic Programming for Symbolic Regression.
+  The Taylor features mined by Taylor polynomial that fits given dataset are used to guide genetic programming.
+  
+paperName: "Taylor Genetic Programming for Symbolic Regression" 
+  GECCO 2022. preprint: https://kgae-cup.github.io/TaylorGP/
+
+  Really hope I win! jk
+url: http://cavalab.org/feat # a link to the project
```

Comment:
change the URL

Reply:
Sorry, I forgot to modify the url in metadata.yml. Thank you for your reminder. Now I have modified it.

Reply:
> change the URL

 Sorry, I forgot to modify the url in metadata.yml. Thank you for your reminder. Now I have modified it.